addons/knobs/README.md

```diff
@@ -37,7 +40,7 @@ Now, write your stories with knobs.
 
 ```js
 import { storiesOf } from '@storybook/react';
-import { withKnobs, text, boolean, number } from '@storybook/addon-knobs';
+import { addonKnobs, text, boolean, number } from '@storybook/addon-knobs/dist/react';
```

Comment:
why did `addonKnobs` come back? is this a bad merge?

Reply:
my bad indeed, good catch